recipes-core/initrdscripts/initramfs-framework/bootfs

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Copyright (C) 2018 Apertus Solutions, LLC
+# Licensed on MIT
+
+bootfs_enabled() {
+    if [ -n "${bootparam_boot}" ]; then
+        info "using boot device ${bootparam_boot}"
+        return 0
+    else
+        debug "no boot device parameter provided"
+        return 1
+    fi
+}
+
+bootfs_run() {
+    BOOTFS_DIR="/${ROOTFS_DIR}/boot/system"
+    if [ ! -d "${BOOTFS_DIR}" ]; then
+            return
+    fi
+
+    if [ -n "${bootparam_boot}" ]; then
```

Comment:
Since bootfs_enabled already checks for bootparam_boot being set, this whole if can be removed?

Reply:
Correct, will fix.